ketcher-autotests/tests/Reaction-tool/Mapping-Unmapping tool/reaction-tools.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { test } from '@playwright/test';
+import {
+  takeLeftToolbarScreenshot,
+  LeftPanelButton,
+  selectLeftPanelButton,
+  selectNestedTool,
+  ReactionMappingTool,
+  ArrowTool,
+  delay,
+  DELAY_IN_SECONDS,
+  openFileAndAddToCanvas,
+} from '@utils';
+
+test.describe('Open Ketcher', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+    await page.waitForSelector('.Ketcher-root');
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeLeftToolbarScreenshot(page);
+  });
+
+  test('Icons and tooltips for Plus Tool', async ({ page }) => {
+    await selectLeftPanelButton(LeftPanelButton.ReactionPlusTool, page);
+  });
+
+  test(' Icons and tooltips for Reaction Mapping tools', async ({ page }) => {
+    await selectNestedTool(page, ReactionMappingTool.MAP);
+    await delay(DELAY_IN_SECONDS.FIVE);
+    await takeLeftToolbarScreenshot(page);
+
+    await selectNestedTool(page, ReactionMappingTool.UNMAP);
+    await delay(DELAY_IN_SECONDS.FIVE);
+    await takeLeftToolbarScreenshot(page);
+
+    await openFileAndAddToCanvas('mapped_rection_benz.rxn', page);
+    await delay(DELAY_IN_SECONDS.FIVE);
+    await selectNestedTool(page, ReactionMappingTool.AUTOMAP);
+  });
+
+  test('Icons and tooltips for Arrow Tools', async ({ page }) => {
+    await selectNestedTool(page, ArrowTool.ARROW_OPEN_ANGLE);
+    await delay(DELAY_IN_SECONDS.FIVE);
```

Comment:
We definitely need to eliminate these delays. I count 17 of them. 17 * 5 = 85 seconds!
And these tests are repeating. My proposal is to add `for` loop. https://playwright.dev/docs/test-parameterize.

Other than that, it is not supposed to be a single test. Let's create a separate test for every icon.